### PR TITLE
Write swagger doc for resources Performance

### PR DESCRIPTION
**Pull request by AndreyLuka**

### CODE REVIEW

File: doc/theatre.json [hunk 1]
```diff
@@ -99,6 +99,160 @@
...
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "query",
+                        "name": "limit",
+                        "description": "Count entries at one page, default is 10",
+                        "required": false,
+                        "type": "integer",
+                        "pattern": "\\d+",
```
Comment (spolischook) [inline doc/theatre.json]: its not needed cause ```"type": "integer"```

File: doc/theatre.json [hunk 2]
```diff
@@ -99,6 +99,160 @@
...
+                        "default": 10
+                    },
+                    {
+                        "in": "query",
+                        "name": "page",
+                        "description": "Number of page to be shown, default is 1",
+                        "required": false,
+                        "type": "integer",
+                        "pattern": "\\d+",
```
Comment (spolischook) [inline doc/theatre.json]: its not needed cause ```"type": "integer"```

File: doc/theatre.json [hunk 3]
```diff
@@ -99,6 +99,160 @@
...
+                        "default": 1
+                    },
+                    {
+                        "in": "query",
+                        "name": "locale",
+                        "description": "Selects language of data you want to receive, default is \"uk\"",
+                        "required": false,
+                        "type": "string",
+                        "pattern": "^[a-zA-Z]+",
```
Comment (spolischook) [inline doc/theatre.json]: replace it to:
```
enum:
  - uk
  - en
```

File: doc/theatre.json [hunk 4]
```diff
@@ -99,6 +99,160 @@
...
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "in": "query",
+                        "name": "locale",
+                        "description": "Selects language of data you want to receive, default is \"uk\"",
+                        "required": false,
+                        "type": "string",
+                        "pattern": "^[a-zA-Z]+",
```
Comment (spolischook) [inline doc/theatre.json]: replace it to:
```
enum:
  - uk
  - en
```

File: doc/theatre.json [hunk 5]
```diff
@@ -99,6 +99,160 @@
...
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "path",
+                        "name": "slug",
+                        "description": "Performance slug",
+                        "required": true,
+                        "type": "string"
+                    },
```
Comment (spolischook) [inline doc/theatre.json]: Add ```"pattern": "^[a-z\-]+$"``` to slug parameter

Comment (AndreyLuka) [inline doc/theatre.json]: what about digits? can we have something like '/performances/post-name-2016'?

Comment (spolischook) [inline doc/theatre.json]: oh, you are right!

Comment (spolischook) [inline doc/theatre.json]: lowercase letters, numbers and hyphens

**Pull request merged**